src/modules/condition/ifchain.rs

```diff
@@ -82,11 +82,11 @@ impl TranslateModule for IfChain {
         let mut is_first = true;
         for (cond, block) in self.cond_blocks.iter() {
             if is_first {
-                result.push(format!("if [ {} != 0 ]; then", cond.translate(meta)));
+                result.push(format!("if [ \"{}\" != 0 ]; then", cond.translate(meta)));
```

Comment:
This shouldn't look like this too. Unless there is a big reason why all the conditions should be wrapped in quotes